Iterators and Closures/Standard Library Types/Arc/src/main.rs

```diff
@@ -14,7 +14,7 @@ fn main() {
                 let mut sum = 0;
                 while i < child_numbers.len() {
                     sum += child_numbers[i];
-                    i += 5;
+                    i += 8;
```

Comment:
The tests do not pass after changing the test
The new sums for offsets are:
Sum of offset 0 is 624
Sum of offset 1 is 637
Sum of offset 2 is 650
Sum of offset 3 is 663
Sum of offset 4 is 576
Sum of offset 5 is 588
Sum of offset 6 is 600
Sum of offset 7 is 612

We should change the hard-coded lines in the test